user-guide/Advanced_Modules/Dashboards_and_Low_Code_Apps/Visualizations/Available_visualizations.md

```diff
@@ -8,279 +8,77 @@ Many visualizations are available in the Dashboards app and Low-Code Apps, in th
-  </div>
-  <div class="column">
-    <a href="/user-guide/Advanced_Modules/Dashboards_and_Low_Code_Apps/Visualizations/Available_visualizations/Tables/Pivot_table_component.html" title="Pivot Table" target="_self"><img src="~/user-guide/images/Pivot_Table.svg" style="width:100%"></a>
-  </div>  
-  <div class="column">
-    <a href="/user-guide/Advanced_Modules/Dashboards_and_Low_Code_Apps/Visualizations/Available_visualizations/Tables/Table_component.html" title="Table" target="_self"><img src="~/user-guide/images/Table.svg" style="width:100%"></a>
-  </div>  
-</div>
+| Visualization | Description |
+|:--:|--|
+| ![Parameter table](~/user-guide/images/Parameter_Table.svg) | A [parameter table](xref:DashboardParameterTable) displays a data table of an element. |
+| ![Alarm table](~/user-guide/images/Alarm_Table.svg) | An [alarm table](xref:DashboardAlarmTable) displays a list of alarms or information events, which can be filtered in multiple ways. |
+| ![Pivot table](~/user-guide/images/Pivot_Table.svg) | A [pivot table](xref:DashboardPivotTable) displays a status report for a number of parameters of the elements using the selected protocol and protocol version. |
+| ![Table](~/user-guide/images/Table.svg) | A [table](xref:DashboardTable) displays the results of a query in a structured, row-column format, similar to a spreadsheet. |
 
 ### Charts
 
-<style>
-.row {
-  display: flex;
-  flex-wrap: wrap;
-  padding: 0 4px;
-}
- 
-/* Create five equal columns that sit next to each other */
-.column {
-  flex: 20%;
-  max-width: 20%;
-  padding: 0 11px;
-}
-</style>
-
-<div class="row"> 
-  <div class="column">
-    <a href="/user-guide/Advanced_Modules/Dashboards_and_Low_Code_Apps/Visualizations/Available_visualizations/Charts/Column_and_Bar_chart.html" title="Column & Bar Chart" target="_self"><img src="~/user-guide/images/Column&Bar_Chart.svg" style="width:100%"></a>
-  </div>
-  <div class="column">
-    <a href="/user-guide/Advanced_Modules/Dashboards_and_Low_Code_Apps/Visualizations/Available_visualizations/Charts/Line_and_area_chart.html" title="Line & Area Chart" target="_self"><img src="~/user-guide/images/Line&Area_Chart.svg" style="width:100%"></a>
-  </div>
-  <div class="column">
-    <a href="/user-guide/Advanced_Modules/Dashboards_and_Low_Code_Apps/Visualizations/Available_visualizations/Charts/Pie_and_donut_chart.html" title="Pie & Donut Chart" target="_self"><img src="~/user-guide/images/Pie&Donut_Chart.svg" style="width:100%"></a>
-  </div>  
-</div>
+| Visualization | Description |
+|:--:|--|
+| ![Column & Bar chart](~/user-guide/images/Column_Bar_Chart.svg) | A [column & bar chart](xref:ColumnAndBarChart) displays the elements or services in a view that caused the most or the least alarms in a selected time range and were in an alarm state for the longest or shortest period of time. |
```

Comment:
This visualization also works with GQI queries so I would refrain from mentioning any specific DataMiner concepts (elements, services, views, alarms, time ranges, etc). It can basically be anything being represented in columns/bars.